internal/common/rules/globalruleregistration.go

```diff
@@ -323,6 +323,36 @@ func registerRulesByMass() {
 			Mutable: false,
 			Linked:  false,
 		},
+		{
+			Name: "increment_budget_speaker",
+			ReqVar: []VariableFieldName{
+				SpeakerIncrement,
+			},
+			Values:  []float64{-1, 100},
+			Aux:     []float64{0},
+			Mutable: true,
+			Linked:  false,
+		},
+		{
+			Name: "increment_budget_judge",
+			ReqVar: []VariableFieldName{
+				JudgeIncrement,
```

Comment:
Rename to JudgeBudgetIncrement for clarity?

Reply:
sure, easy fix